app/Services/RecoveryPasswordService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\ResetPassword;
+use App\Models\User;
+use Carbon\Carbon;
+
+class RecoveryPasswordService
+{
+    public function updatePassword($data)
+    {
+        $recovery = ResetPassword::where('token', $data['token'])->first();
+        $date1 = Carbon::parse($recovery->created_at);
+        $date2 = Carbon::now();
+        if ($date1->diffInMinutes($date2)<2){
```

Comment:
1) we can do something like this:` if (Carbon::now()->diff....($recovery->created_at)) {.....}`
2) use `diffInHours` method